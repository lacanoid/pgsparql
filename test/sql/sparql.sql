select * 
  from sparql.get_properties('dbpedia','http://dbpedia.org/resource/Johann_Sebastian_Bach')
 where predicate='http://dbpedia.org/ontology/birthYear';
