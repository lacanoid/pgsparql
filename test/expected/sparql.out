select * 
  from sparql.get_properties('dbpedia','http://dbpedia.org/resource/Johann_Sebastian_Bach')
 where predicate='http://dbpedia.org/ontology/birthYear';
               predicate               | object | value | lang |                datatype                
---------------------------------------+--------+-------+------+----------------------------------------
 http://dbpedia.org/ontology/birthYear | 1685   | 1685  |      | http://www.w3.org/2001/XMLSchema#gYear
(1 row)

